python/python/lancedb/query.py

```diff
@@ -267,6 +267,16 @@ def to_list(self) -> List[dict]:
         """
         return self.to_arrow().to_pylist()
 
+    def to_batches(self, max_chunksize) -> list:
+        """
+        Execute the query and return the results as a list of dictionaries.
+
+        Each list entry is a dictionary with the selected column names as keys,
+        or all table columns if `select` is not called. The vector and the "_distance"
+        fields are returned whether or not they're explicitly selected.
```

Comment:
I don't know why this says the list items are dictionaries. `self.to_arrow` returns a PyArrow [Table](https://arrow.apache.org/docs/python/generated/pyarrow.Table.html#pyarrow.Table.to_batches), and the `to_batches` method returns a list of `RecordBatch`. Could you update this to be more accurate?

> The vector and the "_distance" fields are returned whether or not they're explicitly selected.

I think we plan on changing the behaviour of this soon. See https://discord.com/channels/1030247538198061086/1030247538667827251/1212101602324914186